client/src/_shared/ActionLink.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button } from 'antd'
+
+export function ActionLink({ onClick, text, classes }) {
+  const handleClick = e => {
+    e.preventDefault()
+    onClick()
+  }
+  return (
+    <>
+      <Button
+        onClick={handleClick}
+        className={`${classes} focus:shadow-none`}
```

Comment:
We might want to check if `classes` is truthy as it is an [optional](https://github.com/pieforproviders/pieforproviders/pull/193/files#diff-a467d5fddf5ec774cd44e196a71c6887R26) prop or set a default value.